Sources/PrinceOfVersions/Objective-C Helpers/Objective-C Extensions/ObjectiveCBaseExtensions.swift

```diff
@@ -78,9 +72,4 @@ internal extension PrinceOfVersions {
                 }
         })
     }
-
-    static func checkHeadersValidity(from headers: NSDictionary?) -> NSError? {
```

Comment:
Please check if ObjC will see setter as NSString:NSString... I think that I had some problems with that and that was the reason for creating this check.